pandas/tests/libs/test_hashtable.py

```diff
@@ -149,6 +156,16 @@ def test_tracemalloc_for_empty(self, table_type, dtype):
             assert get_allocated_khash_memory() == 0
 
 
+def test_get_labels_groupby_for_Int64():
```

Comment:
is this independent of the rest of the PR?

Reply:
I have added this test case to improve the coverage in the fast `test_hashtable.py`, because while refactoring, I've messed things up in `get_labels_groupby` and it was caught by slower/bigger tests.